docs/pages/services/indexer.mdx

```diff
@@ -116,9 +228,10 @@ We can solve this with [volumes](https://docs.docker.com/storage/volumes/):
    **Note:** You should do this every time you restart the blockchain.
    Otherwise your index will include data from multiple blockchains, and make no sense.
 
-#### PostgreSQL
+#### PostgreSQL as storage
 
 The command to start the indexer in PostgreSQL mode is `pnpm start:postgres`.
+This command starts both the indexer and the front end.
```

Comment:
```suggestion
This command starts both the indexer and query frontends.
```

Reply:
We should add a section for starting only the writer or only the query frontend (starting the frontend separately is necessary to scale reads)